src/common/initialize.lua

```diff
@@ -2,6 +2,21 @@ local utils = require(".common.utils")
 local json = require(".common.json")
 local initialize = {}
 
+---@alias InitialANTState {
```

Comment:
Nit: Not sure how well this docs syntax is getting picked up by your tools but what we've used on the network contract is:
```
--- @class InitialAntState
--- @field name string The name
--- @field ticket string The ticker
```
etc.

Reply:
Im using the same tools (i think?) but alias has been working great here. 

Class feels like it indicates methods/metatables to my brain so the alias structure seemed more appropriate for objects.